examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/sensors.py

```diff
@@ -223,7 +223,7 @@ def the_db_connection(init_context):
     get_the_db_connection(init_context.resources.credentials)
 
 
-@sensor(job=the_job)
+@sensor(job=the_job)  # type: ignore (didactic)
```

Comment:
why is this a type error?

Reply:
Because the sensor function does not actually return anything, hence the function arg to the decorator has an invalid type.